src/ViewModel/Converter/AnnualReportTeaserConverter.php

```diff
@@ -3,20 +3,42 @@
 namespace eLife\Journal\ViewModel\Converter;
 
 use eLife\ApiSdk\Model\AnnualReport;
+use eLife\Journal\Helper\DownloadLink;
+use eLife\Journal\Helper\DownloadLinkUriGenerator;
+use eLife\Journal\ViewModel\Paragraph;
+use eLife\Patterns\PatternRenderer;
 use eLife\Patterns\ViewModel;
 use eLife\Patterns\ViewModel\Teaser;
 
 final class AnnualReportTeaserConverter implements ViewModelConverter
 {
+    private $patternRenderer;
+    private $downloadLinkUriGenerator;
+
+    public function __construct(PatternRenderer $patternRenderer, DownloadLinkUriGenerator $downloadLinkUriGenerator)
+    {
+        $this->patternRenderer = $patternRenderer;
+        $this->downloadLinkUriGenerator = $downloadLinkUriGenerator;
+    }
+
     /**
      * @param AnnualReport $object
      */
     public function convert($object, string $viewModel = null, array $context = []) : ViewModel
     {
+        $links = [
+            "<a href=\"{$object->getUri()}\">View report</a>",
+        ];
+
+        if ($object->getPdf()) {
+            $download = $this->downloadLinkUriGenerator->generate(DownloadLink::fromUri($object->getPdf()));
+            $links[] = "<a href=\"{$download}\">Download PDF</a>";
+        }
+
         return Teaser::main(
             $object->getTitle(),
-            $object->getUri(),
-            $object->getImpactStatement()
+            null,
+            ($object->getImpactStatement() ? $this->patternRenderer->render(new Paragraph($object->getImpactStatement())) : '').$this->patternRenderer->render(ViewModel\Listing::forTeaser($links))
```

Comment:
This line's getting a bit confusing, best to extract it. (In case you haven't spotted, `PatternRenderer` takes 1 or more arguments.)